blocks/identity-block/components/social-sign-on/_children/GoogleSignIn.jsx

```diff
@@ -10,31 +10,26 @@ function GoogleSignIn({ customButtons, socialSignOnIn, className }) {
 	const phrases = usePhrases();
 	const googleNotification = phrases.t("identity-block.social-google-one-tap-notification");
 
-	useEffect(() => {
-		if (customButtons) {
-			document.getElementById("custom-google-signin-btn").addEventListener("click", () =>
-				window.google.accounts.id.prompt((notification) => {
-					if (notification.isSkippedMoment()) {
-						// https://developers.google.com/identity/gsi/web/reference/js-reference
-						// https://developers.google.com/identity/gsi/web/guides/features#exponential_cooldown
-						// eslint-disable-next-line
-						alert(googleNotification);
-						// Remove cookie works in Safari
-						document.cookie = `g_state=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT`;
-						window.google.accounts.id.prompt();
-					}
-				}),
-			);
-		}
-	}, [customButtons, googleNotification]);
-
 	if (customButtons) {
 		return (
 			<Button
 				id="custom-google-signin-btn"
 				variant="secondary-reverse"
 				iconLeft={GoogleIcon}
 				className={`${className}__Google`}
+				onClick={() =>
+					window.google.accounts.id.prompt((notification) => {
```

Comment:
Might want to use conditional calling on this in case "google" does not exist.

`window?.google?.accounts?.id?.prompt`